patches.json

```diff
@@ -230,6 +230,9 @@
     "FPC is getting disabled during deployments": {
       ">=2.3.2 <2.3.5": "MAGECLOUD-5069__fpc_is_getting_disabled_during_deployments__2.3.2.patch",
       ">=2.3.5 <2.4.0": "MAGECLOUD-5069__fpc_is_getting_disabled_during_deployments__2.3.5.patch"
+    },
+    "Sitemap Generation Warnings": {
+      ">=2.3.3 <2.3.5": "MCLOUD-3025__sitemap_generation_warnings__2.3.3.patch"
```

Comment:
We have 3 customers who reported about this problem, on that moment they had 2.3.1 and 2.3.3 Magento versions. 
As this problem reproducible only on cloud project, customers on any version may have this issue, but I have concerns about increasing version range.
@YPyltiai your suggestions?

Reply:
@NadiyaS if we know that this reproduces anywhere else beyond this range - we may need to cover that. We don't want to re-release packages next time it happens on 2.3.1. Can we somehow validate it, please?

Reply:
Added 2.3.1 inclusively to cover all active cases

Reply:
That is the problem that we cannot reproduce it. And even on the client projects it is a random error.
This patch does not fix the problem itself. It is a workaround which we do not want to deliver everywhere.
so, we will deliver starting 2.3.1 then?

Reply:
@NadiyaS I honestly see no harm in re-try operation - it will trigger only when initially it failed, which will not affect anyone who has no issues. So I would cover everyone who is affected. At least from my point of view I don't see what else can it break but maybe I am missing something.